.github/dependabot.yml

```diff
@@ -7,3 +7,6 @@ updates:
     directory: "/"
     schedule:
       interval: "weekly"
+    target-branch: "dev"
+    reviewers:
```

Comment:
Is reviewers an option for the dependabot file? I couldn't find it in the documentation, but perhaps I missed it I did find an option for [`assignees`](https://docs.github.com/en/code-security/dependabot/working-with-dependabot/dependabot-options-reference#assignees--) (For assignees, it looks like it may need to be an individual rather than a team)

It looks like for reviewers, they are added through the CODEOWNERS file: https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/customizing-dependabot-prs#automatically-adding-reviewers